projects/components/facet/bootstrap/facet-filters/facet-filters.ts

```diff
@@ -130,4 +99,68 @@ export class BsFacetFilters implements OnChanges {
         return this.facetService.hasData(facet.aggregation, this.results);
     }
 
+    private addFacetMenu() {
+        let outFacets: Action[] = [];
+
+        outFacets.push(new Action({
+            name: `add_remove_all`,
+            text: this.userFacets.length < this.facets.length ? "Add all" : "Remove all",
```

Comment:
One thing that's still missing is the internationalization: I know it's a pain, but could replace all the hard-coded text by message keys and update the message json files accordingly?

Reply:
Sure, I'll do that.